website/i18n/en.json

```diff
@@ -301,6 +301,10 @@
         "title": "Secure Validator",
         "sidebar_label": "Secure Validator"
       },
+      "maintain-guides-society": {
+        "title": "Participate in Society",
```

Comment:
I would consider changing this to "Join Kappa Sigma Mu" since this is currently the only society and it's not clear that there will be more than one in the foreseeable future.